components/tree-select/nz-tree-select.component.ts

```diff
@@ -437,6 +440,9 @@ export class NzTreeSelectComponent implements ControlValueAccessor, OnInit, Afte
 
   ngAfterViewInit(): void {
     this.attachOverlay();
+    if (this.nzTreeSelectTemplate) {
+      setTimeout(() => this.treeRef.nzTreeTemplate = this.nzTreeSelectTemplate, 0);
```

Comment:
Wonder why need `setTimeout`?

Reply:
https://github.com/NG-ZORRO/ng-zorro-antd/blob/92c1a85b2b9f31de124deaa7cfb598ae41cc3bc0/components/tree-select/nz-tree-select.component.ts#L268

I did not realize this line, `setTimeout` is useless.